trunk/SUAVE/Methods/Weights/Buildups/Electric_Stopped_Rotor/empty.py

```diff
@@ -75,14 +75,15 @@ def empty(config,
 # Unpack Inputs
 #-------------------------------------------------------------------------------
 
-    rProp               = config.propulsors.network.propeller.prop_attributes.tip_radius
-    mBattery            = config.propulsors.network.battery.mass_properties.mass
-    mPayload            = config.propulsors.network.payload.mass_properties.mass
+    rPropLift           = config.propulsors.propulsor.propeller_lift.tip_radius
```

Comment:
is this a suave-wide change?  network -> propulsors?

Reply:
It is. I'm going to correct this and delete this pull request per our discussion this morning